src/components/entryList/EntryList.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import useEntry from '../../hooks/useEntry';
+import Entry from '../entry/Entry';
+
+function EntryList() {
+  const { entry } = useEntry();
+
+  return (
+    <ul>
+      {entry.map((entry) => (
+        <li key={`${entry.name}`}>
+          <Entry entry={entry} />
```

Comment:
Your entry should include both the User's username and their message. Currently, it's only displaying the message.